io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,348 @@
+app-id: io.github.pemsley.coot
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+extensions:
+  - org.freedesktop.Platform.GL.nvidia
```

Comment:
```suggestion
```

This isn't a valid manifest property.

Reply:
https://docs.flatpak.org/en/latest/extension.html

I missed the doc and noticed this now.

Reply:
It is the correct format, isn't it?

```
add-extensions:
  org.flatpak.app.plugin:
    version: '1.0'
    directory: extension_directory
    add-ld-path: lib
    merge-dirs: plug-ins;help
    subdirectories: true
    no-autodownload: true
    autodelete: true
```

Reply:
That is yes, but again you don't need to list nvidia explicitly. It gets installed automatically.

Reply:
I read the document, and it seems like `--device=dri` alone is enough.

Reply:
However, I am not sure what lacks in the manifest.

Reply:
@pemsley 

Are you saying that when you tested on NVIDIA GPUs you could not display molecules in Coot, right?

Reply:
> @pemsley
> 
> Are you saying that when you tested on NVIDIA GPUs you could not display molecules in Coot, right?

@pemsley 
How about this?